bufferapp/buffer-analyze
# Pass list of available products to the App Shell

### Purpose

This PR includes the changes to:
- Upgreade `@bufferapp/ui` to 5.34.2.
- Pass a list of `featureFlips` to the `AppShell`.
- If the user has `reply` on the list of `productLinks` then set the `enableReply` feature flip.

### Review

Steps to QA:
1) Login with an account that has access to Analyze and Reply.
2) Confirm that Publish, Analyze and Reply are displayed on the AppShell and all 3 links work.
3) Login with an account that has access to Analyze but it does not have access to Reply.
4) Confirm that only Publish and Analyze are visible on the AppShell.

#### Staging Deployment

_This repo is CI/CD enabled and staging deployment should be available at:
https://<branch_name>.analyze.buffer.com :smile:_
